src/pages/home/report/ReportActionsView.tsx

```diff
@@ -133,6 +133,10 @@ function ReportActionsView({
     }, [route]);
 
     const listID = useMemo(() => {
```

Comment:
@perunt Would you kindly add a comment above the `listID` declaration describing its role and why it is passed down to the FlatList component as (React) `key` instead of passing it to the existing [`extraData`](https://reactnative.dev/docs/flatlist#extradata) array [here](https://github.com/Expensify/App/blob/fed9d6c0f4cb47605233c556ea1794bec002f6f8/src/pages/home/report/ReportActionsList.tsx#L532) ?

I also asked in https://github.com/Expensify/App/issues/38838#issuecomment-2021238263 as part of https://github.com/Expensify/App/issues/38838 (issue).

Reply:
Hey! Definitely, I'll put my comment up top so it's easier to spot. Thanks!

So, here's the deal: I need to make the whole list refresh itself from scratch. It's all about the virtualization thing the list does. Basically, this refresh trick is mostly needed when you're linking comments, especially when jumping from looking at a bunch of messages to just a few. The virtualization in FlatList gets a bit wonky when the amount of stuff shown shrinks as you move to a specific message. So, to keep everything looking right, we gotta refresh the whole list, not just poke the FlatList to update what it's already showing. So if we put it to `extraData`, the comment linking would lead to the wrong place in some specific cases

Reply:
let me check your links, will also leave this info there